### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by LukasMcClelland**
Added `python ci_folder_parsing.py` to checklist

## Description

Added  ci_folder_parsing.py to readme

## Motivation and Context

New and old types of tests should be run

## Impact (If any)

none

## Screenshots:

N/A

## Checklist:

- [x] I have updated the changelog.
- [x] I have updated the documentation (If applicable).
- [x] I have added tests to cover my changes (If applicable).
- [x] All new and existing tests passed.
- [x] All new code passed compilation.

### CODE REVIEW

**Comment by renatoalmeidaoliveira** [comment]
Why not:
`pyats run job folder_parsing_job.py -v -o <os> -c <class>`
That way the test runs only over the modified class and os, and prints the expected and the executed results?

**Comment by tahigash** [comment]
> Why not:
> `pyats run job folder_parsing_job.py -v -o <os> -c <class>`
> That way the test runs only over the modified class and os, and prints the expected and the executed results?

@renatoalmeidaoliveira you're right. to test only added parser, that's the best syntax. but here we are trying to ask to test everything **just in case**.

@LukasMcClelland but technically, we are doing both `python -m unittest -v` and `python ci_folder_parsing.py` via GitHub Actions. so we might not need to ask contributor to do this anymore.

File: README.md [hunk 1]
```diff
@@ -51,8 +51,7 @@ YouTube Video: <How to write a Genie parser for Cisco!> https://youtube.com/watc
 Once you create a new parser, don't forget to check 1, 2, and 3;
 if you only update the parser class without modifying/creating the schema, please check 2 and 3.
```
Comment (tahigash) [inline README.md:52]: need to update here, it says 1,2 and 3... and also please check other location as well to adjust to this change.

Let's add 3 like:
```
- [ ] 3. make sure GitHub Actions checks passed.

Comment (LukasMcClelland) [reply README.md:52]: Fixed

**Review by tahigash — APPROVED** [review]

**Comment by tahigash** [comment]
@LukasMcClelland please get 2nd approval on this.

**Review by jeaubin — APPROVED** [review]

**Pull request merged**